lib/commands/connect/auth_connection.js

```diff
@@ -34,13 +34,13 @@ module.exports = {
             callbackServer();
 
             try {
-              let args = {'environment':'production', 'next': 'http://127.0.0.1:18000'};
+              var args = {'environment':'production', 'next': 'http://127.0.0.1:18000'};
```

Comment:
I'll admit, I noticed this one after I said they were all function-level. This is actually bad code anyway, and should be rewritten to avoid putting the assignment inside the `try` block at all, but that's a separate issue.